cats/src/main/scala/magnolify/cats/auto/package.scala

```diff
@@ -16,4 +16,59 @@
 
 package magnolify.cats
 
-package object auto extends LowPriorityImplicits
+import cats._
+import cats.kernel.{Band, CommutativeGroup, CommutativeMonoid, CommutativeSemigroup}
+
+// set implicit priority to avoid conflicts
+// see: https://typelevel.org/cats/guidelines.html#implicit-instance-priority
+// use shapeless.LowPriority so the
+// provided cats type classes are always preferred
+// triggers derivation as last resort
+package object auto extends LowPriority0Implicits
+
+trait LowPriority0Implicits extends LowPriority2Implicits {
```

Comment:
I wonder how this worked before the fix. Wrong type class was picked up by compiler?

Reply:
I think the `shapeless.LowPriority` got our back and was giving one of the implicit depending on the resolution.